mapviz/include/mapviz/mapviz_plugin.h

```diff
@@ -27,12 +27,21 @@
 //
 // *****************************************************************************
 
-#ifndef MAPVIZ_MAPVIZ_PLUGIN_H_
-#define MAPVIZ_MAPVIZ_PLUGIN_H_
+#ifndef MAPVIZ__MAPVIZ_PLUGIN_H_
+#define MAPVIZ__MAPVIZ_PLUGIN_H_
 
-// C++ standard libraries
-#include <string>
+// ROS libraries
+#include <swri_transform_util/transform.h>
+#include <swri_transform_util/transform_manager.h>
+#include <rclcpp/rclcpp.hpp>
+#include <tf2/transform_datatypes.h>
+#include <tf2_ros/buffer.h>
+#include <tf2_ros/transform_listener.h>
 
+#include <mapviz/widgets.h>
+#include <yaml-cpp/yaml.h>
+
+// Boost libraries
 #include <boost/make_shared.hpp>
 #include <boost/shared_ptr.hpp>
```

Comment:
I saw in one of the other files a transition from boost pointers to std pointers; should we be trying to clean up other references to boost pointers as well?

Reply:
That's a good catch, we should avoid using boost::shared_ptr anywhere that we can get away with it now (which should be everywhere).  I'll scan over the files and get rid of references to these.